src/Tizen.NUI.Scene3D/src/public/Controls/Panel.cs

```diff
@@ -0,0 +1,377 @@
+        /// </summary>
+        /// <returns>True if this model receives shadow.</returns>
+        private bool IsShadowReceiving()
+        {
+            var isShadowReceiving = Interop.Panel.IsShadowReceiving(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return isShadowReceiving;
+        }
+
+        /// <summary>
+        /// Sets defined resolution to the Panel.
+        /// The resolution is independent from the Panel size property.
+        /// The resolution defines a plane that the 2D UI scene will be rendered.
+        /// And the shape of the panel plane is defined by aspect ratio of the input resolution.
+        /// </summary>
+        /// <param name="resolution">The resolution defines panel plane.</param>
+        /// <returns>True if this model receives shadow.</returns>
+        private void SetPanelResolution(Vector2 resolution)
+        {
+            Interop.Panel.SetPanelResolution(SwigCPtr, Vector2.getCPtr(resolution));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Retrieves panel resolution.
+        /// </summary>
+        /// <returns>Resolution of the panel plane.</returns>
+        private Vector2 GetPanelResolution()
+        {
+            Vector2 resolution = new Vector2(Interop.Panel.GetPanelResolution(SwigCPtr), true);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return resolution;
+        }
+
+        /// <summary>
+        /// Sets root Actor of 2D UI content.
+        /// The content is rendered on the plane of the Panel by using off screen rendering.
+        /// </summary>
+        /// <param name="content">Root View of 2D UI content.</param>
+        private void SetContent(View content)
+        {
+            Interop.Panel.SetContent(SwigCPtr, content.SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Retrieves root View of 2D UI content.
+        /// </summary>
+        /// <returns>Root View of 2D UI content.</returns>
+        private View GetContent()
+        {
+            IntPtr cPtr = Interop.Panel.GetContent(SwigCPtr);
+            View ret = Registry.GetManagedBaseHandleFromNativePtr(cPtr) as View;
+            if (ret != null)
+            {
+                NUI.Interop.BaseHandle.DeleteBaseHandle(new HandleRef(this, cPtr));
+            }
+            else
+            {
+                ret = new View(cPtr, true);
+            }
+            NDalicPINVOKE.ThrowExceptionIfExists();
+            return ret;
+        }
+
+        /// <summary>
+        /// To make transitionSet instance be disposed.
+        /// </summary>
+        protected override void Dispose(DisposeTypes type)
```

Comment:
```suggestion
        [EditorBrowsable(EditorBrowsableState.Never)]
        protected override void Dispose(DisposeTypes type)
```

Add this keyword, to make clean ACR PR in future

Reply:
Thank you.